code/query.sql

```diff
@@ -701,4 +701,16 @@ create table cb_dev_groots.retailer_leads(
   updated_at timestamp default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP ,
   updated_by int(11) not null,
   primary key (id)
+)ENGINE=InnoDB default CHARSET=utf8;
+
+alter table groots_employee drop column department_id;
+
+create table cb_dev_groots.employee_department_mapping(
+  id int(11) not null AUTO_INCREMENT,
+  employee_id int (11) not null,
+  department_id int(11) not null,
+  created_at datetime not null,
+  updated_at timestamp default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP ,
+  updated_by int(11) not null,
+  primary key (id)
```

Comment:
add foreign keys
change table name to employee_departments
send api changes also as mapping has changed